test/cases/ostree-simplified-installer.sh

```diff
@@ -102,24 +102,29 @@ SSH_KEY_PUB=$(cat "${SSH_KEY}".pub)
 # kernel-rt package name (differs in CS8)
 KERNEL_RT_PKG="kernel-rt"
 
+# Only RHEL 9.2 and above support user customizations in simplified installer blueprint
 case "${ID}-${VERSION_ID}" in
     "rhel-8.8")
         OSTREE_REF="rhel/8/${ARCH}/edge"
         OS_VARIANT="rhel8-unknown"
+        USER_IN_SIMPLIFIED="false"
```

Comment:
just note that if this change will get into 9.2 before ITM26, then it will end up in 8.8 as well.